plugins/source/plugin.go

```diff
@@ -47,7 +47,7 @@ const (
 func addInternalColumns(tables []*schema.Table) {
 	for _, table := range tables {
 		if c := table.Column("_cq_id"); c != nil {
-			continue
+			panic(fmt.Sprintf("table %s already has column _cq_id", table.Name))
```

Comment:
I seem to remember a case where exactly this happened, but can't remember what the situation was now. Maybe I'm imagining it :)

Reply:
I think we will know given it will be panic :)